research/flamby/utils.py

```diff
@@ -99,6 +99,8 @@ def evaluate_model_on_dataset(
                 preds = model(input, personal=True)["personal"]
             else:
                 preds = model(input)
+                if isinstance(preds, tuple):
```

Comment:
I don't think `preds` should be coming back as a tuple, but maybe I'm missing something. It should always be a dictionary now?